platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheProblemReportingIntegrationTest.groovy

```diff
@@ -367,6 +367,10 @@ class ConfigurationCacheProblemReportingIntegrationTest extends AbstractConfigur
             problemsWithStackTraceCount = 1
         }
 
+        // The interrupting problem cause must not be repeated in the aggregated CC build failure
+        // TODO: introduce a structured failure assert instead of checking all error output: https://github.com/gradle/gradle/issues/33985
```

Comment:
💅 `TODO:configuration-cache`

Reply:
This issue is not CC specific, why does it need a CC qualifier?